operation/fhir-operation-reindex/src/main/java/com/ibm/fhir/operation/reindex/ReindexOperation.java

```diff
@@ -148,8 +148,9 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
                             String[] parts = resourceLogicalId.split("/");
                             rt = parts[0];
                         }
+                        // Check resource type
                         if (!ModelSupport.isConcreteResourceType(rt)) {
-                            throw FHIROperationUtil.buildExceptionWithIssue("ResourceType is not valid", IssueType.INVALID);
+                            throw FHIROperationUtil.buildExceptionWithIssue("Resource type '" + rt + "' is not valid", IssueType.INVALID);
                         }
```

Comment:
is it safe to remove this now that we're checking it in AbstractOperation?

Reply:
never mind, forgot that we still allow users to pass it in via that logicalResourceId param